cloudbridge/cloud/base/provider.py

```diff
@@ -37,20 +39,26 @@ def default_result_limit(self):
         :rtype: ``int``
         :return: The maximum number of results to return
         """
+        log.debug("Maximum number of results for list methods "
```

Comment:
pylint recommends passing debug parameters as follows:
log.debug("Maximum number of results for list methods %s", DEFAULT_RESULT_LIMIT)